assets/prefabs/world/Shrine.prefab

```diff
@@ -1,4 +1,7 @@
 {
   "alwaysRelevant": true,
-  "Shrine": {}
+  "Shrine": {},
+  "Health": {
```

Comment:
Like would this be regular health, or GooeyDefense health? Just looking at the diff leaves me wondering

Reply:
GooeyDefence, like above I shan't depend on core. Thus there will only be one health system.